packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -11,6 +11,7 @@ interface ContentProps {
 		src: ImageProps['src'];
 		alt?: ImageProps['alt'];
 	};
+	contentClassName?: string;
```

Comment:
Does it have to be a prop on the component? Or can we just add the class to our component straight up?

Reply:
In react we need to do this, I'm not the biggest fan of it, but is needed to pass external classes as props

Reply:
ah I see. It doesn't seem to be showing up on the in the snapshot.

Reply:
It won't appear in the snapshot because the join doesn't add empty spaces on undefined values

Reply:
Oh yeah meakes sense on the snapshot in `nextjs-kit`, I meant the snapshots in the `next-wordpress-starter`, is that expected?

Also, the more I think about this the more I'm ok with it as it adds a way to style the content with custom classes a bit easier.

Reply:
The changes on snapshots are here https://github.com/pantheon-systems/decoupled-kit-js/pull/358/files#diff-b8b0b846e64924ebb479741da6e4d519b05d538bf9cee22e7114495d8a9065b8R223 👀